exercises/dominoes/dominoes.rb

```diff
@@ -0,0 +1,53 @@
+class Dominoes
+  require_relative 'domino'
+  attr_accessor :list
+
+  #[ArrayTuples], false => [Domino] or [Domino], true => [Domino]
+  def initialize( arr=[], already_dominoes= nil)
+    @list = []
+    arr.each do |item|
```

Comment:
That is much cleaner, thanks.
I'd still actually like that @list because I believe I'm going to need to construct separate sublists and keep track of them for the ultimate n-domino solution.



> On Sep 1, 2016, at 1:56 PM, Tute Costa <notifications@github.com> wrote:
> 
> In exercises/dominoes/dominoes.rb:
> 
> > @@ -0,0 +1,53 @@
> > +class Dominoes
> > +  require_relative 'domino'
> > +  attr_accessor :list
> > +
> > +  #[ArrayTuples], false => [Domino] or [Domino], true => [Domino]
> > +  def initialize( arr=[], already_dominoes= nil)
> > +    @list = []
> > +    arr.each do |item|
> Rubyism: if you use map instead of each, you don't need the @list variable.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>